src/Connector.SqlServer/Connector/SqlServerConnector.cs

```diff
@@ -78,44 +77,64 @@ public Task<string> GetCorrelationId()
             string correlationId, DateTimeOffset timestamp, VersionChangeType changeType,
             IDictionary<string, object> data)
         {
-            try
-            {
-                // If we are in Event Stream mode, append extra fields
-                var dataToUse = new Dictionary<string, object>(data);
-                if (StreamMode == StreamMode.EventStream)
+            var tableName = new SanitizedSqlName(containerName);
+            var dataToUse = new Dictionary<string, object>(data)
                 {
-                    dataToUse.Add(TimestampFieldName, timestamp);
-                    dataToUse.Add(ChangeTypeFieldName, changeType);
-                    dataToUse.Add(CorrelationIdFieldName, correlationId);
-                }
-                else
-                    dataToUse.Add(TimestampFieldName, timestamp);
+                    { TimestampFieldName, timestamp }
+                };
+
+            // If we are in Event Stream mode, append extra fields
+            if (StreamMode == StreamMode.EventStream)
+            {
+                dataToUse.Add(ChangeTypeFieldName, changeType);
+                dataToUse.Add(CorrelationIdFieldName, correlationId);
+            }
 
+            try
+            {
                 if (_bulkSupported)
                 {
                     var bulkFeature = _features.GetFeature<IBulkStoreDataFeature>();
                     await bulkFeature.BulkTableUpdate(
                         executionContext,
                         providerDefinitionId,
-                        containerName,
+                        tableName,
```

Comment:
When I was introducing schema parameter, my thoughts were that in terms of working with SQL we have schemas and tables. So these terms would be easier to read and understand by developers looking into this code.
Because it's not obvious what are schema and containerName, how to couple this pair.